vehicle/autoware_raw_vehicle_cmd_converter/src/node.cpp

```diff
@@ -215,6 +241,41 @@ void RawVehicleCommandConverterNode::onControlCmd(const Control::ConstSharedPtr
   control_cmd_ptr_ = msg;
   publishActuationCmd();
 }
+
+void RawVehicleCommandConverterNode::onSteering(const Steering::ConstSharedPtr msg)
```

Comment:
Please mention that this subscriber can not be replaced by `take()` method in a comment.

Reply:
Originally it was take(), in which case the topic name must be stated in the declaration. In that case, there would be two identical topics. Therefore, subscriber, which does not require a topic name to be specified at the time of declaration, is used.
In any case, add a comment to the code.

![image](https://github.com/user-attachments/assets/e469f54b-9b9f-4d95-a11a-3940dd6ddf3c)

Reply:
> In that case, there would be two identical topics.

I mean, this node also need to publish steering topic if converting option is true.

![image](https://github.com/user-attachments/assets/4493f256-48b0-493f-ada0-6c585d47ac4d)

Reply:
added comment why not use polling sub

https://github.com/autowarefoundation/autoware.universe/pull/8504/commits/5b667ccf9f6052a37c888062a623273f2eaf4254